docs/README.md

```diff
@@ -19,23 +19,23 @@ The initial images will need to be packed into the initial tasks image, and then
 
 The root server ELF image will need to be manipulated appropriately to include a packed memory image (as well as other data structures).
 
-## seL4 Core Platform Library
+## Microkit Library
 
-The header and source files for the library are in the `libsel4cp` directory.
+The header and source files for the library are in the `libmicrokit` directory.
 
 To compile against the library you need flags such as:
 
-    -I$(LIBSEL4CP)/include -L$(LIBSEL4CP) -llibsel4cp
+    -I$(LIBMICROKIT)/include -L$(LIBMICROKIT) -llibmicrokit
```

Comment:
`-l` prepends `lib`, so it should be -lmicrokit

Reply:
Yes good point, I will make a separate PR for fixes as this is supposed to just be a straight find and replace kind of patch.

I've noted some other issues I found as well which can be in the same subsequent PR.